Modules/Devices/src/Devices.ConsumerApi/Controllers/IdentitiesController.cs

```diff
@@ -110,7 +110,7 @@ public async Task<IActionResult> GetDeletionProcesses(CancellationToken cancella
     }
 
     [HttpPut("Self/DeletionProcesses/{id}/Cancel")]
-    [ProducesResponseType(typeof(HttpResponseEnvelopeResult<CancelDeletionProcessResponse>), StatusCodes.Status200OK)]
+    [ProducesResponseType(typeof(HttpResponseEnvelopeResult<CancelDeletionProcessAsOwnerResponse>), StatusCodes.Status200OK)]
```

Comment:
The route should be `/Identity/{address}/DeletionProcesses/{deletionProcessId}/Cancel`

Reply:
Given that the controller's name is `IdentitiesController` the path can be either:
`/Identities/Identity/{address}...`
`/Identities/{address}/...`

I went with the second option both with this endpoint as well as with `CancelDeletionProcessAsSupport`.

Changes made.

Reply:
> Given that the controller's name is IdentitiesController the path can be either:
/Identities/Identity/{address}...
/Identities/{address}/...
> Which option should be taken? 

Sorry, my bad. It's `Identities` of course.

> Passing in both identity address and a deletion process ID would require reworking of the command and handler as well. Confirm that you are requesting this.

Confirmed.

Reply:
Changes made.

Reply:
Damn, it seems I placed my comment on the wrong controller. In case of the consumer api the route shouldn't contain the address of course. That doesn't make sense.

Reply:
Changes made.